tests/data/format/final_period/exotic_line_ending.py.out

```diff
@@ -0,0 +1,13 @@
+def function():
+    """Check the name of first argument, expect:
+
+      *'self' for a regular method
+      *'cls' for a class method or a metaclass regular method
+    """
+
+def function():
+    """This has peculiar line ending;
+
+    but it should not matter.
+
```

Comment:
I think this could be considered a minor issue, the spaces at the end were transformed to a newline instead of just removed.

Reply:
This sounds like a `Stripper` issue. We can't just strip all empty lines, but an empty line as last line should be strippable.

Can you open an issue?

Reply:
Opened #54